admin/src/main/java/edu/unc/lib/dl/admin/controller/PerformanceMonitorController.java

```diff
@@ -0,0 +1,334 @@
+	};
+	private CSVFormat csvFileFormat = CSVFormat.DEFAULT.withRecordSeparator(NEW_LINE_SEPARATOR);
+	
+	@Autowired
+	private String dataPath;
+	
+	@Autowired
+	private JedisPool jedisPool;
+	
+	public JedisPool getJedisPool() {
+		return jedisPool;
+	}
+
+	public void setJedisPool(JedisPool jedisPool) {
+		this.jedisPool = jedisPool;
+	}
+
+	public Set<String> getDepositMetrics() {
+		try (Jedis jedis = getJedisPool().getResource()){
+			return jedis.keys("deposit-metrics:*");
+		}
+	}
+	
+	private Boolean buildFile(String path) {
+		File csvFile = new File(path);
+		if (csvFile.exists() && !csvFile.isDirectory()) { 
+			Long currentTime = System.currentTimeMillis();
+			Long fileCreationTime = csvFile.lastModified();
+
+			if ((currentTime - fileCreationTime ) > MILLISECONDS_IN_ONE_HOUR) {
+				return true;
+			}
+			return false;
+		}else if (!csvFile.exists()) {
+			return true;
+		} else {
+			return false;
+		}
+	}
+	
+	/**
+	 * Gets totals for metrics that are only aggregated daily such as moves & enhancements
+	 * It also grabs throughput totals for older data, before these fields were set to measure performance at the individual deposit level
+	 * @return
+	 */
+	public String getOperationsData() {
+		String filePath = dataPath + "ingest-times-daily.csv";
+
+		if (buildFile(filePath)) {
+			Set<String> deposits = getDepositMetrics();
+			try (Jedis jedis = getJedisPool().getResource()) {
+				FileWriter fileWriter = null;
+				CSVPrinter csvFilePrinter = null;
+				Set<String> operations = null;
+				Map<String, String> depositJob = null;
+				Map<String, String> operationJob = null;
+				String[] depositKeys = null;
+				
+				operations = jedis.keys("operation-metrics:*");
+
+				fileWriter = new FileWriter(filePath);
+				csvFilePrinter = new CSVPrinter(fileWriter, csvFileFormat);
+				csvFilePrinter.printRecord(FILE_HEADERS);
+
+				Boolean matchingDate = false;
+				for (String deposit : deposits) {
+					depositKeys = deposit.split(":");
+					
+					// Ignore data for individual deposits by uuid. Only need the daily ones in this instance
+					if (depositKeys.length > 2) {
+						continue;
+					}
+					
+					depositJob = jedis.hgetAll(deposit);
+
+					String jobDate = depositKeys[1];
+					String throughputFiles = depositJob.get("throughput-files");
+					String throughputBytes = depositJob.get("throughput-bytes");
+					String finished = depositJob.get("finished");
+					String failed = depositJob.get("failed");
+					String failedDepositJob = depositJob.get("failed-job:edu.unc.lib.dl.cdr.services.techmd.TechnicalMetadataEnhancementService");
+
+					for (String operation : operations) {
+						String operationDate = operation.split(":")[1];
+						
+						if (operationDate.equals(jobDate)) {
+							operationJob = jedis.hgetAll(operation);
+
+							String moves = (operationJob.get("moves"));
+							String imageEnhancements = (operationJob.get("finished-enh:edu.unc.lib.dl.cdr.services.imaging.ImageEnhancementService")); 
```

Comment:
I would suggest making an array of enhancement class names, and then looping through them to when adding to the data instead of hardcoding a series of variables like this.  It'll be a pain when we add more classes to have to keep counting all the zeroes to add